conductor-requirements.txt

```diff
@@ -1,4 +1,5 @@
-git+https://github.com/ansible/ansible.git@v2.3.0.0-0.1.rc1#egg=2.3.3.0
+git+https://github.com/ansible/ansible.git@v2.3.0.0-0.1.rc1#egg=ansible-2.3.0.0
```

Comment:
We can change this over to devel again, now that the `synchronize` bug we were hitting is fixed.